LocalStorage.coffee

```diff
@@ -124,6 +124,19 @@ class LocalStorage extends events.EventEmitter
         if e.code != "EEXIST"
           throw e
       return
+  sync:()->
```

Comment:
Thanks for the pull request!!!

Please add a blank line above like all the other methods.

Also, please rename to `_sync` since it's not part of the standard localStorage API